api-2.6/src/test/java/org/openmrs/module/fhir2/api/search/EncounterSearchQueryImpl_2_6Test.java

```diff
@@ -93,11 +93,11 @@ public void searchForEncounters_shouldReverseIncludeMedicationRequestsAndAssocia
 		
 		assertThat(results, notNullValue());
 		
-		assertThat(resultList.size(), equalTo(10)); // reverse included resources added as part of the result list
```

Comment:
there were two orders of type "DISCONTINUE" in the core test data set, so there are a lot of test cases where the expected result is reduced by 2